### ISSUE TITLE
PWA Bootstrap 5 upgrade

### ISSUE THREAD

**Issue by JoeArmani** [episphere/connect#1167]
PWA: Bootstrap 5 upgrade (from Bootstrap 4)
---
https://getbootstrap.com/docs/5.0/getting-started/introduction/
https://getbootstrap.com/docs/5.0/getting-started/accessibility/

---
DevOps determined these upgrades and compatibilities are important alongside the rollout of Quest 2.
• Bootstrap 4 is out of date. Bootstrap 5 is required for security compliance.
• Bootstrap 5 offers enhanced accessibility and speed, and some additional features compared to Bootstrap 4.
• Quest, the survey module loaded into the PWA, uses Bootstrap 5, and we should make these Bootstrap versions compatible.
• Bootstrap 5 removes the jQuery dependency.

This update retains the PWA's existing visual aspects, with upgraded semantics for accessibility and Bootstrap 5 compatibility.

The following components received updates and should be tested on desktop, tablet, and mobile-sized screens:

1. Participant sign-up form
2. Participant verification form
3. Navigation menu
4. Dashboard
5. Messages page
6. Forms page
   -- HIPAA revocation form
   -- Data destruction form
7. My Profile page
  -- Name update
  -- Contact information update
  -- Mailing address update
  -- Sign In Information update
9. My Payment page
10. My Samples page
11. All pop-up modals. These will be encountered while navigating through the above list.
12. Email and phone login

Additional test: please check for duplicate login emails. This PR takes the first step to resolve https://github.com/episphere/connect/issues/1156

Additional test: please go to My ToDo List (surveys page), turn on VoiceOver, and use the keyboard to navigate to the survey start and continue buttons. VoiceOver should read the survey names. Refresh the page after toggling languages and repeat. Note: VoiceOver adds the word 'Button' in English based on screen reader settings regardless of the selected language in MyConnect. This tests PR https://github.com/episphere/connectApp/pull/864, where the screen reader was reading the language markup syntax instead of the survey name when accessing a survey button

**Comment by JoeArmani** [comment]
This PR is in review.

**Comment by JoeArmani** [comment]
This is in dev and ready to test

**Comment by robertsamm** [comment]
@JoeArmani is VoiceOver specific to apple products? My PC is a Dell but I have an iPhone and iPad I can test this with. Is there something else I should be using for the PC?

**Comment by JoeArmani** [comment]
@robertsamm Yes, that's exactly right: VoiceOver is specific to Apple products.

TL;DR the JAWS/Windows implementation is not ready to test this cycle...
JAWS is the PC version, but that requires a license. I'm waiting on a PC to fully test JAWS for My ToDo List survey entry and surveys. It's a nuanced issue because I've been developing in a Virtual Terminal (Windows) with a Mac, but the Mac and PC keyboards are slightly different, and that causes issues with JAWS keyboard navigation.

**Comment by JoeArmani** [comment]
Yes, please just navigate to the Survey buttons and make sure the announcement is intelligible. The screen reader (VoiceOver & JAWS) announcement was missed when languages were implemented, so I noticed the survey buttons were reading out the development markup that was used instead of the actual survey names, which was confusing for accessibility.

The in-survey accessibility testing is a separate item and will take substantial time, I think, so this is just about testing those buttons.

**Comment by robertsamm** [comment]
@JoeArmani 
(1) I have tested all of the above bootstrap items in dev on PC, iphone, and iPad and they looked good. Ready for stage!
(2) For the two magic link emails - none of my 3 test cases received 2 magic links at the same time and I had to log in and out a couple of times for each just bc of timing out. I also tried clicking 'continue' twice, is there any other scenarios to test for this?
(3) For the VoiceOver testing I have tested this on iPhone and iPad in dev and looked good. @cunnaneaq is going to do a quick test on a Mac computer when she does one of the cancer screening survey test participant scenarios

**Comment by JoeArmani** [comment]
@robertsamm That was a ton of testing. I appreciate you getting to it so quickly.
(1) Excellent.
(2) Nope, that's perfect, and you covered all the test cases.
(3) Great to hear. We'll wait for confirmation from @cunnaneaq

Thanks again!

**Comment by cunnaneaq** [comment]
@JoeArmani and @robertsamm worked well on a MacBook in Chrome

**Comment by JoeArmani** [comment]
This is tested in dev and ready for stage.

Note: https://github.com/episphere/connectApp/pull/877, mentioned above, will not hold up the stage push, but we'll confirm that testing for dev and stage by tomorrow.

**Comment by robertsamm** [comment]
@JoeArmani tested in stage on PC, iPhone, iPad. Ready for prod!

**Comment by JoeArmani** [comment]
Closing - in prod
